internal/cmd/organization/command.go

```diff
@@ -0,0 +1,26 @@
+package organization
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+)
+
+type command struct {
+	*pcmd.AuthenticatedStateFlagCommand
+}
+
+func New(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "organization",
+		Short:       "View Confluent Cloud organizations.",
```

Comment:
You can update an org so it's not just `viewing`?

Reply:
Good point; changed it to `Manage your ...`